apps/electron/layers/main/src/events/register.ts

```diff
@@ -0,0 +1,28 @@
+import { app,BrowserWindow } from 'electron';
```

Comment:
How does this line pass the lint & format?

Reply:
No idea 🥵